packages/editor/src/client/panel/peerProfileEditPanel.tsx

```diff
@@ -47,6 +51,12 @@ export function PeerInfoEditPanel({onClose, onSave}: ImageEditPanelProps) {
       setLogoImage(data.peerProfile.logo)
       setName(data.peerProfile.name)
       setThemeColor(data.peerProfile.themeColor)
+      setCallToActionText(
+        data?.peerProfile?.callToActionText?.length
```

Comment:
you don't need the `?` here. The GraphQL ensures that you don't have `null` here and you can just set the CallToActionText the same way as `setThemeColor`